src/selectors/profile.js

```diff
@@ -154,6 +155,17 @@ export const getContentfulSpeedIndexProgress: Selector<
 export const getProfilerConfiguration: Selector<?ProfilerConfiguration> = state =>
   getMeta(state).configuration;
 
+export const getActiveBrowsingContextID: Selector<BrowsingContextID | null> = state => {
+  const configuration = getProfilerConfiguration(state);
+  if (configuration) {
+    // BrowsingContext ID can be `0` and that means Firefox has failed to get
+    // the BrowsingContextID of the active tab. We are converting that `0` to
+    // `null` here to explicitly indicate that we don't have that information.
+    return configuration.activeBrowsingContextID || null;
```

Comment:
Thanks for the explanation here. I'm still a bit nervous using truthy values to compute this. I'd be more comfortable with explicit checks.